UI/src/asset-type-layer-specifications.js

```diff
@@ -49,7 +49,8 @@
         showUnit: true
       },
       label: new LinearAssetLabel(),
-      hasTrafficSignReadOnlyLayer: true
+      hasTrafficSignReadOnlyLayer: true,
+      hasVerificationButton: true
```

Comment:
hasVerificationButton could be changed to isVerifiable or something more descriptive